src/audio_to_image_model/model_audio_to_image.py

```diff
@@ -74,11 +107,12 @@ def _test(dataloader, model, loss_fn):
     TRAIN_SET_FILE_DIR = r"\train_audio"
 
     # Load the metadata file
-    metadata_df, num_classes = load_metadata(r"../../data/local_subset.csv")
+    # metadata_df, num_classes = load_metadata(r"../../data/local_subset.csv")
+    metadata_df, num_classes = load_metadata(r"../../data/local_subset_stratified.csv")
 
     # Create a custom dataset object
     augmentations = AugmentMelSpectrogram()
-    bird_dataset = BirdDataset(metadata_df, DATASET_BASE_FILE_PATH + TRAIN_SET_FILE_DIR, fixed_length=300, num_classes=num_classes, transform=augmentations or None)
+    bird_dataset = BirdDataset(metadata_df, DATASET_BASE_FILE_PATH + TRAIN_SET_FILE_DIR, num_classes=num_classes, transform=augmentations, efficientnet=True)
```

Comment:
This is an example of something we should not track in code but better in a config.json or something. Here only configuration is changed and not functionality

Reply:
I added a configuration dictionary, but as a Python dict